### PR TITLE
Final touches

### LINKED ISSUE #242: Add directory checker to make sure rhubarb and ffmpeg is installed
**Is this related to a problem? Please describe.**
Nodejs can load and run our app without rhubarb or ffmpeg installed, and it doesn't always print those errors to terminal

**Describe the solution you'd like**
check directories first with some error handling

### LINKED ISSUE #269: get tooling documentation updates done
We need official documentation for the tooling

### LINKED ISSUE #290: Tooling documentation requires screenshot of illustrator layer setup
Just a simple screenshot 4 da docs :) @vnajack I believe you're the only one who still has access to illustrator so if you could do this that would be greatly appreciated!!

### LINKED ISSUE #297: Add maximum time for avatar eyes to stay open
**Is this related to a problem? Please describe.**
In Peer Testing 2, we found in one of the clips that the avatar didn't blink at all

**Describe the solution you'd like**
Add a maximum time for avatar eyes to stay open so that our avatars don't get dry eyes

### PR DESCRIPTION
**Pull request by vnajack**
- moved files to improve directory organization
- added max time for avatar to NOT blink (don't want their little eyes to get dry)
- polished README and added links to documentation, screenshots, and Adobe Illustrator script
- added error explanations for when Rhubarb and ffmpeg are not installed correctly
- added links to example output videos
- closes #297 
- closes #290 
- closes #269 
- closes #242

### CODE REVIEW

**Review by mkuelker — APPROVED** [review]
Looks good! Nice work!

**Review by mkuelker — APPROVED** [review]
Looks good! Nice work!

**Review by mkuelker — APPROVED** [review]
Looks good! Nice work!

File: README.md [hunk 1]
```diff
@@ -148,14 +151,18 @@ could lead to lip synchronization errors.
 | 403 Forbidden | Request is recognized by the server but refused. Likely due to connecting to the wrong port |
 | 500 Internal Server Error | The API has not been configured properly on the host server machine |
 
+If Rhubarb is not installed correctly or fails to read the audio and text files, the output may include a file not found error for `./rhubarb`, an `exit code (status): null`, and a file not found error for the .MP4 output file. Please ensure Rhubarb is installed correctly (refer to [setup](#setup)), check that the audio file is a valid .WAV file, and that the text file is a valid .txt file.
```
Comment (edouarde1) [inline README.md:154]: Great! this is really good that we document this :)

File: README.md [hunk 2]
```diff
@@ -203,11 +210,19 @@ The [frame data output](#frame-data-output) includes a path to the image and the
 video. This file is then sent to FFMPEG, a highly portable library capable of rendering image frames into an mp4
 output. Once the video file is created, it is then returned to the user as a response from the API.
 
-## Example
+## Output Examples
```
Comment (edouarde1) [inline README.md:213]: sweet!!

**Review by edouarde1 — APPROVED** [review]
great additions to the documentation with the error codes and output contents

**Pull request merged**